iset.mm

```diff
@@ -37961,8 +37961,19 @@ this axiom (that is, Non-wellfounded Set Theory but in the absence of
   ${
     $d A x y $.
     $( No class is a member of itself.  Exercise 6 of [TakeutiZaring] p. 22.
+
+       The reason that this theorem is marked as discouraged is a bit subtle.
+       If we wanted to reduce usage of ~ ax-setind , we could redefine
+       ` Ord A ` ( ~ df-iord ) to also require ` _E Fr A ` ( ~ df-frind ) and
+       in that case any theorem related to irreflexivity of ordinals could use
+       ~ ordirr (which under that definition would presumably not need
+       ~ ax-setind to prove it).  But since ordinals have not yet been defined
+       that way, we cannot rely on the "don't add additional axiom use" feature
+       of the minimizer to get theorems to use ~ ordirr .  To encourage
+       ~ ordirr when possible, we mark this theorem as discouraged.
+
```

Comment:
Can you add to the comment of ordirr something like:
The present proof requires ax-setind.  If in the definition of ordinals df-iord, we also required that membership be well-founded on any ordinal (see df-frind), then we could prove ordirr without ax-setind.

Reply:
I'd approve a pull request to add that.

Reply:
So would I.  I think the addition of this comment to elirr entails the addition of a similar comment to ordirr, and the person who does one does the other.

Reply:
OK, OK, time for both of us to take a step back and laugh at the absurdity of the situation where we have a wording, we both agree on it, and we're trying to argue about who should put it in a pull request. I'll put the following in my next pull request:

```diff
diff --git a/iset.mm b/iset.mm
index e1d46bc0..2690d6ab 100644
--- a/iset.mm
+++ b/iset.mm
@@ -37983,8 +37983,11 @@ $)
   $}
 
   $( Epsilon irreflexivity of ordinals: no ordinal class is a member of itself.
-     Theorem 2.2(i) of [BellMachover] p. 469, generalized to classes.
-     (Contributed by NM, 2-Jan-1994.) $)
+     Theorem 2.2(i) of [BellMachover] p. 469, generalized to classes.  The
+     present proof requires ~ ax-setind .  If in the definition of ordinals
+     ~ df-iord , we also required that membership be well-founded on any
+     ordinal (see ~ df-frind ), then we could prove ~ ordirr without
+     ~ ax-setind .  (Contributed by NM, 2-Jan-1994.) $)
   ordirr $p |- ( Ord A -> -. A e. A ) $=
     ( wcel wn word elirr a1i ) AABCADAEF $.
 
```

Reply:
Thanks.  I think it was indeed up to you, but don't worry, I'm taking that very lightly.

Reply:
Taking it lightly seems like good advice.